cmake/compileroptions.cmake

```diff
@@ -48,19 +48,19 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wfloat-equal")              # Floating values used in equality comparisons
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Winline")                   # If a inline declared function couldn't be inlined
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wmissing-declarations")     # If a global function is defined without a previous declaration
-    set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wmissing-format-attribute") # 
+    set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wmissing-format-attribute") #
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Woverloaded-virtual")       # when a function declaration hides virtual functions from a base class
-    set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wpacked")                   # 
+    set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wpacked")                   #
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wredundant-decls")          # if anything is declared more than once in the same scope
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wshadow")                   # whenever a local variable or type declaration shadows another one
-    set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wsign-promo")               # 
+    set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wsign-promo")               #
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wno-missing-field-initializers")
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wno-missing-braces")
     set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wno-sign-compare")
 
     if(WARNINGS_ANSI_ISO)
-#        set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Werror=return-type")        # 
-#        set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Wstrict-aliasing=3")
+        #        set(EXTRA_C_FLAGS "${EXTRA_C_FLAGS} -Werror=return-type")        #
```

Comment:
if you move the bracket it's not necessary to remove the "set" also

Reply:
I did not change this intentionally - it was done automatically by my IDE and somehow some of these changes do not show up in the Diffs of it...will try to revert them.